style.css

```diff
@@ -330,7 +331,104 @@ li {
   color: #f55800;
 }
 
-.button:disabled {
-  background-color: #f4f1f0;
-  color: #5b5957;
+.form {
+  height: 844px;
+  background-color: #fff;
+  display: flex;
+  min-width: 375px;
+  flex-direction: column;
+  gap: 58px;
+  padding: 120px 24px 120px 24px;
+  justify-content: center;
+}
+
+/* .text-container {
+  min-width: 327px;
+  height: 216px;
+  display: flex;
+  align-items: center;
+} */
```

Comment:
Kindly remove all commented-out code from your project🙏. Commented code makes your codebase look unclean and messy and doesn't adhere to the [DRY](https://www.techtarget.com/whatis/definition/DRY-principle) principle of coding